.github/workflows/pr_change_check.yml

```diff
@@ -22,7 +22,9 @@ jobs:
         run: |
           pr_url="https://github.com/${{ github.repository }}/pull/${{ github.event.number }}"
           echo $pr_url
-          gh pr diff $pr_url --name-only > $HOME/changed_files
+          gh pr checkout $pr_url --name-only > $HOME/changed_files
```

Comment:
Would it be better to change the checkout sha above to
```
      with:
        ref: refs/pull/${{ github.event.pull_request.number }}/merge
        fetch-depth: 2 # include merge commit parents
```

which should allow direct diff without needing to checkout again?